great-idea/index.html

```diff
@@ -15,9 +15,79 @@
 </head>
 
   <body>
-
+    <header>
+      <nav>
+        <a href="#">Services</a>
```

Comment:
In order to navigate to your services page you should replace the # with the file path. In this case a simple `services.html` is all you need